README.md

```diff
@@ -469,15 +476,15 @@ rather unlikely to occur (enclosed with square brackets).
 
 ### Common
 
-During (re-)synchronisation status messages are shown. Also dropped
-Superframes or AU are mentioned.
+During (re-)synchronisation status messages are shown. Furthermore, dropped
+Superframes or AUs are mentioned.
 
 If the Reed Solomon FEC was used to correct bytes of a Superframe, this
 is mentioned by messages of the format `(3+)` in cyan color. This
-shorter format is used as those messages occure several times with
+shorter format is used as those messages occur several times with
 borderline reception. The digit refers to the number of corrected bytes
 within the Superframe while a plus (if present) indicates that at least
-one byte was incorrectable.
+one byte was uncorrectable.
```

Comment:
Actually "incorrectable" is already valid...

Reply:
According to my (poor) English knowledge, the word **incorrectable** makes no sense and might not exist at all.

Don't forget that the word **correct** could be a verb or an adjective.

Of course the word **incorrect** (which is an adjective) does exist, but it cannot get the suffix -able.

See https://en.wiktionary.org/wiki/uncorrectable

But I'm not an English native speaker ...

Reply:
The word seems to be rarely used, but a lack of a Wiktionary article doesn't mean that it doesn't exist. The search engine of your choice returns several results (e.g. [here](https://dict.tu-chemnitz.de/english-german/incorrectable.html)) and despite that I'm pretty sure that anyone reading to word will be able to understand its meaning. So I currently don't see any strong reason to change this.